### ISSUE TITLE
Allow testing non-catalog fonts

### ISSUE THREAD

**Issue by davelab6** [googlefonts/gfregression#38]
There are some fonts available from `fonts.googleapis.com/css?family=FAMILY` but not from `https://fonts.google.com/download?family=FAMILY` (which is only for font available from `https://fonts.google.com/specimen/FAMILY`)

GFR should be updated with an try/except or if/else so that when `download?family=FAMILY` fails it double checks `css?family=FAMILY&subset=ALL` using a blank user agent string in the HTTP GET request, to obtain the URLs of TTFs from `gstatic.com`

**Comment by m4rc1e** [comment]
Thanks for filing this.

I'm curious if `fonts.googleapis.com/css?family=FAMILY` and `https://fonts.google.com/download?family=FAMILY` are in sync?

We had this issue filed to google/fonts recently, https://github.com/google/fonts/issues/1679

**Comment by davelab6** [comment]
1679 was a bug, can be ignored.

They are in sync if the latter is available.

**Comment by m4rc1e** [comment]
I just realised we may not be use the api endpoint. This endpoint is for web fonts. I think they've all had some form of post processing (removed OT features etc). I'll need to test this first.

**Comment by davelab6** [comment]
SUBSET=ALL should help reduce the delta.

**Comment by m4rc1e** [comment]
Yeah I'll download a font with curl/wget so I get the unsubsetted fonts. I'll then diff it against the downloadable versions from the gf site.

**Comment by m4rc1e** [comment]
I just downloaded a font using wget from the css endpoint and diffed it against the downloadable family on fonts.google.com. Unfortunately, they're very different. Api fonts lack most of the OT features. I think they're still being run through pyftsubset which is then stripping out these features.


![screen shot 2018-09-17 at 14 03 01](https://user-images.githubusercontent.com/7525512/45624663-deb94a00-ba82-11e8-95d3-79767dc47d42.png)
